### DISCUSSION TITLE
AD: naver.com

### DISCUSSION THREAD

**Discussion by gminshin** [List-KR/community#33] [General]
### URL

www.naver.com

### FILTERS

list-kr
list-kr dns

### Browser

Google Chrome

### Operating System

Microsoft Windows 11

### Ad-Blocker

AdGuard for Windows

### Comment

애드가드 사용 후 네이버가 너무 느립니다.

특이 증상으로는 네이버 블로그 이미지 로딩 속도가 엄청 느리네요..

**Comment by piquark6046** [comment]
아래 비디오처럼 2초이내로 페이지가 로드됩니다.
혹시 다른 안티-바이러스나 금융 보안 프로그램을 설치하셨나요?

[Screencast from 2023-04-07 12-40-24.webm](https://user-images.githubusercontent.com/98787049/230610723-3d25f838-596a-4c86-a06f-131e7240c818.webm)

**Reply by gminshin** [reply]
List-KR 필터가 작년 12월 것이어서 발생하는 문제였던 것으로 보입니다. 해결 완료했습니다.
